### PR TITLE
Minor fixes and cleanups to the docs

### PR DESCRIPTION
**Pull request by timleslie**
A handful of minor paper-cuts being fixed up.

### CODE REVIEW

File: docs-next/pages/guides/document-fields.mdx [hunk 1]
```diff
@@ -490,6 +490,8 @@ export default ({ children }) => {
       <DocumentFeaturesProvider>
         <div className="prose">
           <H1>How To Use Document Fields</H1>
+          <p>The [`document`](../apis/fields#document) field type provides a highly configurable rich content field based on [Slate](https://docs.slatejs.org/).
```
Comment (timleslie) [inline docs-next/pages/guides/document-fields.mdx:493]: @mitchellhamilton I'd love to include an introductory paragraph like this (exact wording TBD) before dropping people into the editor example with no other context. Is there an easy/correct way do that with markdown content, or would have we have write raw react/html for this? Links and formatting quickly becomes a bit fraught at that point.

Comment (mitchellhamilton) [reply docs-next/pages/guides/document-fields.mdx:493]: Without https://github.com/keystonejs/keystone/blob/5691466a6b30bf59d0a3f8d0a0fd0d4bd94c6210/docs-next/lib/prose-lite.ts#L1 being completely removed and replaced with using the MDXProvider, the only other option would be to have another mdx file for this bit of content and then import it and render it here.

Comment (timleslie) [reply docs-next/pages/guides/document-fields.mdx:493]: Importing from another mdx file sounds less gross than writing raw react, let me give that a shot and see what it looks like.

Comment (timleslie) [reply docs-next/pages/guides/document-fields.mdx]: Yep, that works pretty cleanly, thanks 👍

File: docs-next/pages/guides/_doc-field-intro.mdx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The [`document`](../apis/fields#document) field type provides a highly configurable rich content field based on [Slate](https://docs.slatejs.org/).
+The Admin UI includes a document editor, which allows content creators to quickly and easily add content to your system.
+You can try out the document editor below.
+
```
Comment (timleslie) [inline docs-next/pages/guides/_doc-field-intro.mdx:4]: Happy to take edits to this content if the wording here isn't what we want.

File: docs-next/lib/initialDocumentDemoContent.ts [hunk 3]
```diff
@@ -240,7 +240,7 @@ export const initialContent = [
     children: [
       {
         text:
-          'Everything above this line 👇 is editable. Expand the block below to see how the data is stored.',
+          'This is the end of the editable document. Expand the block below to see how the data is stored.',
```
Comment (timleslie) [inline docs-next/lib/initialDocumentDemoContent.ts:243]: Pointing down to refer to stuff above is a bit of a brain-bender, this is hopefully clearer.

File: docs-next/pages/apis/config.mdx [hunk 4]
```diff
@@ -160,7 +160,7 @@ export default config({
 import type { ServerConfig } from '@keystone-next/types';
 ```
 
-The **run** commands from the Keystone [command line](../guides/cli) will start an Express web-server for you.
+The `dev` and `start` commands from the Keystone [command line](../guides/cli) will start an Express web-server for you.
```
Comment (timleslie) [inline docs-next/pages/apis/config.mdx:163]: We no longer frame the CLI in groups of commands, so **run** no longer means anything.

**Review by mitchellhamilton — APPROVED** [review]

**Pull request merged**